.github/workflows/build.yml

```diff
@@ -18,8 +18,8 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v1
         with:
-          node-version: 12
-      - run: yarn --frozen-lockfile
+          node-version: 18
+      - run: yarn
```

Comment:
Curious to why we are removing the `--forzen-lockfile` considering this is intended to be 1-1 to what people are using?
This shouldn't be trying to update things at all.

Reply:
Added back.  Given this is for the workflow builds, it should still be there; I don't remember why I removed it.